ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh

```diff
@@ -137,6 +138,7 @@ if [[ "$BASE_OP" != "$META_OPERATOR" ]]; then
   # Build openstack-operator bundle and index
   export IMAGENAMESPACE=${ORGANIZATION}
   export IMAGEREGISTRY=${REGISTRY}
+  export IMAGEBASE=${SERVICE_NAME}
```

Comment:
Discussed with douglas, we don't have underscore in IMAGEBASE

Reply:
Fixed, thanks Sandeep.